orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js

```diff
@@ -176,4 +176,69 @@ module('Integration | Component | routes/authenticated/campaign/assessment/list'
       assert.contains('En attente de participants');
     });
   });
+
+  module('when the campaign doesn‘t have badges', function() {
+    test('it should not display badge column', async function(assert) {
+      // given
+      const campaign = store.createRecord('campaign', {
+        id: 1,
+        name: 'campagne 1',
+        idPixLabel: 'identifiant externe',
```

Comment:
**suggestion** : on peut supprimer `idPixLabel`, ici dans notre test `it should not display badge column`, ce n'est pas ce qu'on veut vérifier, ça apporte que du bruit

Reply:
Done 👍